src/app/app.component.spec.ts

```diff
@@ -5,7 +5,10 @@ import { AppComponent } from './app.component';
 import { StoreModule } from '@ngrx/store';
 import { RouterTestingModule } from '@angular/router/testing';
 
-import { RouterOutletStubComponent } from '../testing';
+import { Component } from '@angular/core';
+
+@Component({selector: 'router-outlet', template: ''})
+export class RouterOutletStubComponent { }
```

Comment:
Earlier this was written in separate files in the view that this component can be used by multiple files, but as it turns out `ng compiler` in recent version update does not allow any component written outside `NgModule` so I moved it here, in AppComponent spec, as `ng build --prod` was erroring here.